### PR TITLE
Fix header button alignment, update content layout

### PR DESCRIPTION
**Pull request by darkwing**
Let me know how you like this.

### CODE REVIEW

File: landoui/templates/partials/layout.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
 {% include "partials/navbar.html" %}
 <section class="hero is-light is-bold">
   <div class="container">
-    <p class="subtitle">{% block banner_text %}{% endblock %}</p>
+    <div class="subtitle"></div>
```
Comment (purelogiq) [inline landoui/templates/partials/layout.html]: Do you not like the little banner? It is redundant currently on the sign out page, but, I feel it adds some character to the site and could be useful once the signout page has more information. 

Also if you wanted to remove it you have to remove it from all the other pages.

Comment (purelogiq) [inline landoui/templates/partials/layout.html]: An alternative would be just a colored stripe, which could be what you have done already (haven't seen it)

Comment (darkwing) [inline landoui/templates/partials/layout.html]: I think that gradient is a nice top nav border but I don't like it as a page heading -- seems a bit Web 2.0-ish.

**Comment by darkwing** [comment]
This is ready.

File: landoui/templates/partials/layout.html (hunk 1 above)
Comment (purelogiq) [inline landoui/templates/partials/layout.html]: To clarify, if you remove this line in the layout, then you have to remove it from the revision.html page as you have already done in the signout page :)

Comment (darkwing) [inline landoui/templates/partials/layout.html]: Sorry, misunderstood.  Updated.

File: landoui/assets_src/css/lando.scss [hunk 2]
```diff
@@ -24,6 +24,17 @@ body {
...
+  padding: 1em 0;
+}
+
+h1 {
+  font-size: 2em;
+  font-weight: bold;
+  margin-bottom: .25em;
+}
+
+p {
```
Comment (purelogiq) [inline landoui/assets_src/css/lando.scss]: This style seems awfully generic. For example on the revisions page, the section headings are really tall for no reason.

# Before 
before2_p_margin

# After
after_p_margin

**Pull request merged**